src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -98,7 +98,7 @@ public static ProjectContextBuilder CreateBuilder(string projectPath, NuGetFrame
         /// <summary>
         /// Creates a project context for each framework located in the project at <paramref name="projectPath"/>
         /// </summary>
-        public static IEnumerable<ProjectContext> CreateContextForEachFramework(string projectPath, ProjectReaderSettings settings = null, IEnumerable<string> runtimeIdentifiers = null)
+        public static IEnumerable<ProjectContext> CreateContextForEachFramework(string projectPath, ProjectReaderSettings settings = null, IEnumerable<string> runtimeIdentifiers = null, IEnumerable<string> dependencyFragments = null)
```

Comment:
Should I propagate the fragments to all the create methods, or just the ones that build and publish use? The fragment scenario is specific to VS builds only.